qubes-rpc/admin.vm.TerminalDispVM

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash --
+
+set -eu
+sock_name="$(mktemp -u XXXXXX)"
```

Comment:
Better use random directory in `/tmp` and socket within it. `/var/run/` is for runtime data, not temporary files/sockets.